database/migrations/2022_03_27_123425_create_module_features_table.php

```diff
@@ -18,6 +18,7 @@ public function up()
             $table->integer('module_id');
             $table->string('name');
             $table->boolean('is_done')->default(false);
+            $table->index(['id', 'module_id']);
             $table->timestamps();
```

Comment:
please add column `description` for this table

Reply:
done